src/api/notices.js

```diff
@@ -0,0 +1,19 @@
+import http from "k6/http";
+import { logResult } from "../common/dynamicScenarios/utils.js";
+import { buildDefaultParams } from "../common/envVars.js";
+import { getBaseUrl, getInnerBaseUrl } from "../common/environment.js";
+
+export const NOTICES_API_NAMES = {
+  notices: "notices",
```

Comment:
let's keep the structure "domain/apiName"
apiName could be the operationId of the openApi, while the domain could the tag, or the controller
let's use uppercase: it should be intended as a constant